### ISSUE TITLE
Production planning with product sequence

### ISSUE THREAD

**Issue by ng-galien** [axelor/axelor-business-suite#3519]
Problem when planning a manufacturing order which create a product with a sequence  

The NPE is
```
2019-04-16 22:34:30.084 ERROR 48943 --- [nio-8080-exec-7] c.a.exception.service.TraceBackService   : java.lang.NullPointerException
	at com.axelor.apps.stock.service.StockLocationLineServiceImpl.findIncomingPlannedStockMoveLines(StockLocationLineServiceImpl.java:628)
	at com.axelor.apps.stock.service.StockLocationLineServiceImpl.computeFutureQty(StockLocationLineServiceImpl.java:590)
	at com.axelor.apps.stock.service.StockLocationLineServiceImpl.updateLocation(StockLocationLineServiceImpl.java:377)
	at com.axelor.apps.supplychain.service.StockLocationLineServiceSupplychainImpl.updateLocation(StockLocationLineServiceSupplychainImpl.java:55)
	at com.axelor.apps.stock.service.StockLocationLineServiceImpl.updateDetailLocation(StockLocationLineServiceImpl.java:279)
	at com.google.inject.persist.jpa.JpaLocalTxnInterceptor.invoke(JpaLocalTxnInterceptor.java:62)
	at com.axelor.apps.stock.service.StockLocationLineServiceImpl.updateLocation(StockLocationLineServiceImpl.java:88)
	at com.google.inject.persist.jpa.JpaLocalTxnInterceptor.invoke(JpaLocalTxnInterceptor.java:62)
	at com.axelor.apps.stock.service.StockMoveLineServiceImpl.updateLocations(StockMoveLineServiceImpl.java:740)
	at com.axelor.apps.supplychain.service.StockMoveLineServiceSupplychainImpl.updateLocations(StockMoveLineServiceSupplychainImpl.java:253)
	at com.axelor.apps.stock.service.StockMoveLineServiceImpl.updateLocations(StockMoveLineServiceImpl.java:458)
	at com.axelor.apps.stock.service.StockMoveServiceImpl.updateLocations(StockMoveServiceImpl.java:323)
	at com.axelor.apps.stock.service.StockMoveServiceImpl.plan(StockMoveServiceImpl.java:296)
	at com.axelor.apps.supplychain.service.StockMoveServiceSupplychainImpl.plan(StockMoveServiceSupplychainImpl.java:158)
	at com.google.inject.persist.jpa.JpaLocalTxnInterceptor.invoke(JpaLocalTxnInterceptor.java:62)
	at com.axelor.apps.production.service.manuforder.ManufOrderStockMoveService.createToProduceStockMove(ManufOrderStockMoveService.java:202)
	at com.axelor.apps.production.service.manuforder.ManufOrderWorkflowService.plan(ManufOrderWorkflowService.java:128)
	at 

```

I solve the issue with some changes in StockLocationLineServiceImpl.java to select the detailed stock location.
<img width="1299" alt="Capture d’écran 2019-04-16 à 23 39 46" src="https://user-images.githubusercontent.com/33391039/56246092-5004be00-60a1-11e9-8d61-e65c3bae3a7a.png">

**Comment by ale-axelor** [comment]
Hello and thank you for your feedback,
This is currently an issue in the latest version (5.1.5). It is fixed in dev branch by this commit : bb0f2e6d9

**Comment by ng-galien** [comment]
Thank you.
